media/windows/calendar-reservations/calendar-reservations.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+#
+# You need all the Python packages in requirements.txt.
+#
+# You probably want to use a Python virtual environment for this project.
+# For example:
+#
+# $ virtualenv --python=python3.8
+# $ . ./venv/bin/activate
+# $ pip3 install `cat requirements.txt`
+#
+
+import sys
+sys.path.insert(0, '../../../python')
+
+import ECC
+import Google
+import GoogleAuth
+
+import datetime
+
+from oauth2client import tools
+
+default_app_json  = 'gcalendar-reservations-client-id.json'
+default_user_json = 'user-credentials.json'
+
+verbose = True
+debug   = False
+logfile = None
+
+# dictionary of calendars that we're checking for events on
+calendars = [
+    { 
+        "name" : "Test calendar #1",
+        "id": 'c_bj96menjelb4pecracnninf45k@group.calendar.google.com',
+    },
+    {
+        "name" : "Test calendar #2",
+        "id" : 'c_ncm1ib261lp6c02i46mors4isc@group.calendar.google.com',
+    },
+    {
```

Comment:
Until we're ready / have high confidence that everything will work out, it might be good to leave this one commented out.